src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -115,6 +115,22 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
                     Opacity(1.0);
                 }
             }
+            else if (viewModelProperty == L"AntialiasingMode")
+            {
+                _NotifyChanges(L"CurrentAntiAliasingMode");
+            }
+            else if (viewModelProperty == L"CloseOnExit")
+            {
+                _NotifyChanges(L"CurrentCloseOnExitMode");
+            }
+            else if (viewModelProperty == L"BellStyle")
+            {
+                _NotifyChanges(L"IsBellStyleFlagSet");
+            }
+            else if (viewModelProperty == L"ScrollState")
+            {
+                _NotifyChanges(L"CurrentScrollState");
+            }

```

Comment:
This got moved over from `Profiles.cpp`.

This is step 2 of my PR description.